tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -131,32 +131,51 @@ TMP_LOGDIR=$(mktemp -d --suffix _tensorboard_logdir)
 tensorboard --port="${TEST_PORT}" --logdir="${TMP_LOGDIR}" &
 TB_PID=$!
 
-TEST_URL="http://localhost:${TEST_PORT}/data/logdir"
 echo
 echo "tensorboard binary should be running at pid ${TB_PID}"
-echo "Sending test HTTP requests at URL: ${TEST_URL} (${NUM_RETRIES} retries)"
-echo
-
-RETRY_COUNTER=0
-REQUEST_SUCCEEDED=0
-while [[ "${RETRY_COUNTER}" -lt "${NUM_RETRIES}" ]]; do
-  sleep 1
-  STATUS_CODE=$(curl -Is "${TEST_URL}" | head -1 | cut -d ' ' -f 2)
-  if [[ "${STATUS_CODE}" == 200 ]]; then
-    echo
-    echo "Request to ${TEST_URL} succeeded (200)!"
-    echo
-    REQUEST_SUCCEEDED=1
-    break
-  else
-    : $(( RETRY_COUNTER++ ))
-    echo "Request to ${TEST_URL} failed. Will retry in 1 second..."
+echo
+
+function test_access_url(){
+  # Test requests to a URL with curl.
```

Comment:
How about something more descriptive, like "Attempt to fetch the given URL repeatedly until an HTTP 200 response is returned, failing if this does not occur within $NUM_RETRIES requests separated by a one-second delay."?

(Google-style also wants you to document which globals are used and modified, declaring other variables as `local`, so: `local test_url="{$1}"`, etc.)